discretize/View.py

```diff
@@ -737,7 +737,7 @@ def plotGrid(
     def plot_3d_slicer(self, v, xslice=None, yslice=None, zslice=None,
                        vType='CC', view='real', axis='xy', transparent=None,
                        clim=None, aspect='auto', grid=[2, 2, 1],
-                       pcolorOpts=None, fig=None):
+                       pcolorOpts=None, showIt=False, fig=None):
```

Comment:
The `showIt` should not be necessary here. Please see the discussion we had in the previous pr: https://github.com/simpeg/discretize/pull/157#discussion_r280660843

Reply:
Sure. That is fine.

Reply:
It looks like the fix for now is to return figure instead of automatically plotting